Clinical-Genomics/cgweb
# hide add sample button until lab order type selected

How to test:
0. Install branch on stage on clinical-db
1. Open order portal on stage
2. Log in
3. Press "New Order"

Expected outcome:
No Add Sample button visible
The add button should become visible if you select a type other than "Scout" or "externally sequenced" and press Save
